03-git-github-and-markdown/COURSE_MATERIAL.md

```diff
@@ -1307,10 +1635,30 @@ Once the pull request is approved, you will receive an invitation to join the
 GitHub organization. Accept the invitation so you are part of the course and we
 can notify all the students at the same time using GitHub Discussions.
 
+> [!IMPORTANT]
+>
+> You must accept the invitation to be part of the GitHub organization. If you
+> do not accept the invitation, you will not receive notifications from the
+> teaching staff when announcements are made.
+
 #### Check the results
 
 Once the pull request is approved and merged, you can check the results on the
-organization README.
+organization README. You should see your name in the list of students! You can
+now fetch the changes to your local repository.
+
+#### Change the membership visibility of the organization (optional)
+
+If you want, you can change the visibility of your membership in the GitHub
+organization. You can choose between public and private.
+
+You can do so by going in the settings of the organization and then in the
+**Membership** tab.
+
+This will allow other users to see your membership in the organization. They
+will be able to see your username and the date you joined the organization.
+
+This is totally optional. You can keep your membership private if you want.
```

Comment:
I would need to do some tests with you @hadrylouis to validate this part whenever you are available.